analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SecurityPInvokeMethodShouldNotBeCalled.CSharp12.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
```

Comment:
S3884 is deprecated, I would not lose time and add/review/maintain UTs for a rule that will be deleted in the near future. Let me know what you think, I did not review it yet.